docs/software/exportformat.md

```diff
@@ -61,8 +62,33 @@ are still included in reporting to provide access to historic data. Deprecated c
 | Referring Tool                           | For users who were linked to the absentee tool by another tool, the name of the tool                                                                                                                                                                 |
 | Updated At (UTC)                         | The date and time when this action's data was last updated, in UTC                                                                                                                                                                                   |
 
+## Calendar tool export format
```

Comment:
Just alphabetizing the tools here (we renamed this from "Upcoming" to "Calendar" previously, but it was still positioned as if it started with a U)